integration/docker/csi/alluxio/nodeserver.go

```diff
@@ -258,6 +234,27 @@ func getAndCompleteFusePodObj(nodeId string, req *csi.NodeStageVolumeRequest) (*
 	return csiFusePodObj, nil
 }
 
+func (ns *nodeServer) createFusePodIfNotExist(fusePod *v1.Pod) error {
+	if _, err := ns.client.CoreV1().Pods(os.Getenv("NAMESPACE")).Create(fusePod); err != nil {
+		if strings.Contains(err.Error(), "already exists") {
+			glog.V(4).Infof("Fuse pod %s already exists. Skip creating pod.", fusePod.Name)
+		} else {
+			return status.Errorf(codes.Internal, "Failed to launch Fuse Pod at %v.\n%v", ns.nodeId, err.Error())
+		}
+	}
+	return nil
+}
+
+func checkIfMountPointReady(mountPoint string) error {
+	command := exec.Command("bash", "-c", fmt.Sprintf("cat /proc/mounts | grep %v | grep alluxio-fuse", mountPoint))
+	_, err := command.Output()
+	if err != nil {
+		glog.V(3).Infoln("Mount point is not ready, or error occurs while checking.", err.Error())
+		return status.Error(codes.Internal, fmt.Sprintf("Mount point is likely not ready, or error occurs while checking.\n%v", err.Error()))
```

Comment:
Why need to log and return error, since the error message is the same as the log message, i would expect end-user see the error message and no need for a duplication info message

Reply:
Makes sense. Since this whole process is before the application pod is ready, I don't think we can actually grab the logs on the client side. So logging actually makes more sense.

Reply:
More specifically, these returned values can only be found in `kubelet` logs, which very likely can't be seen by end-users.

Reply:
oh, then logging make sense